lib/src/streams/stream_handler.dart

```diff
@@ -67,9 +67,10 @@ class Http2StreamImpl extends TransportStream
   // Termination handler. Invoked if the stream receives an RST_STREAM frame.
   void Function(int) _onTerminated;
 
-  final Function _canPushFun;
-  final Function _pushStreamFun;
-  final Function _terminateStreamFun;
+  final ZoneUnaryCallback<bool, Http2StreamImpl> _canPushFun;
+  final ZoneBinaryCallback<ServerTransportStream, Http2StreamImpl, List<Header>>
+      _pushStreamFun;
+  final ZoneUnaryCallback<dynamic, Http2StreamImpl> _terminateStreamFun;
```

Comment:
Guess this is because we still have this bug/issue with the new `bool Function(bool, Http2StreamImpl)` syntax?

Reply:
Yup. If you want to keep supporting 1.24...